air/src/aux_table/bitwise/mod.rs

```diff
@@ -407,3 +488,40 @@ pub fn agg_bits<E: FieldElement>(row: &[E], start_idx: usize) -> E {
     }
     result
 }
+
+// CYCLE MASKS
+// ================================================================================================
+pub const BITWISE_POW2_K0_MASK: [Felt; OP_CYCLE_LEN] = [
+    Felt::ONE,
+    Felt::ZERO,
+    Felt::ZERO,
+    Felt::ZERO,
+    Felt::ZERO,
+    Felt::ZERO,
+    Felt::ZERO,
+    Felt::ZERO,
+];
+
+pub const BITWISE_POW2_K1_MASK: [Felt; OP_CYCLE_LEN] = [
+    Felt::ONE,
+    Felt::ONE,
+    Felt::ONE,
+    Felt::ONE,
+    Felt::ONE,
+    Felt::ONE,
+    Felt::ONE,
+    Felt::ZERO,
+];
```

Comment:
as noted above, these constant names should be updated

Reply:
Done! Please see [abb346a](https://github.com/maticnetwork/miden/pull/289/commits/abb346aa50875a16ebd6eb30f793475f20faf9ff).